lib/routes/prototype-admin-routes.js

```diff
@@ -104,6 +104,14 @@ router.post('/password', function (req, res) {
   }
 })
 
+router.use((req, res, next) => {
```

Comment:
its important that this middleware comes before any Manage Prototype routes we want to block, would it help to make that clear in a code comment?

Reply:
I've added a comment as requested